experiments/HSA_20150714_MI/hsa_20150714_MI.py

```diff
@@ -0,0 +1,286 @@
+
+# Define source solutions in the vial holder
+#TODO : Define solutions once prepared with the Quantos. Enter compound mass and solvent mass values. - MI
+
+hsa_solution = SimpleSolution(compound=hsa, compound_mass=10.05014 * ureg.milligram, solvent=buffer, solvent_mass=3.4245 * ureg.gram, location=PipettingLocation(
+    source_plate.RackLabel,
+    source_plate.RackType,
+    1))
+
+aspirin_solution = SimpleSolution(compound=aspirin, compound_mass= 20 * ureg.milligram, solvent=buffer, solvent_mass= 10 * ureg.gram, location=PipettingLocation(
+    source_plate.RackLabel,
+    source_plate.RackType,
+    2))
+
+naproxen_sodium_solution = SimpleSolution(compound=naproxen_sodium, compound_mass=25.245 * ureg.milligram, solvent=buffer, solvent_mass=10.0728 * ureg.gram, location=PipettingLocation(
+    source_plate.RackLabel,
+    source_plate.RackType,
+    3))
+
+
+drugs = [aspirin, naproxen_sodium]
+drug_solutions = [aspirin_solution, naproxen_sodium_solution]
+drug_kas = [aspirin_ka, naproxen_ka]
+
+
+
+# Define ITC protocol.
+
+# Protocol for 'control' titrations (water-water, buffer-buffer,
+# titrations into buffer, etc.)
+
+control_protocol = ITCProtocol(
+    'control_protocol',
+    sample_prep_method='Chodera Load Cell Without Cleaning Cell After.setup',
+    itc_method='ChoderaWaterWater5.inj',
+    analysis_method='Control',
+    experimental_conditions=dict(target_temperature=25, equilibration_time=60, stir_rate=1000, reference_power=5),
+    injections=[
+        dict(volume_inj=0.1, duration_inj=0.4, spacing=60, filter_period=0.5)] +
+        5 * [dict(volume_inj=1.5, duration_inj=6, spacing=120, filter_period=0.5)],
+    )
+
+
+blank_protocol = ITCProtocol(
+    '1:1 binding protocol',
+    sample_prep_method='Chodera Load Cell Without Cleaning Cell After.setup',
+    itc_method='ChoderaHSA20.inj',
+    analysis_method='Onesite',  # Protocol for 1:1 binding analysis
+    experimental_conditions=dict(target_temperature=25, equilibration_time=300, stir_rate=1000, reference_power=5),
+    injections=[
+        dict(volume_inj=0.1, duration_inj=0.4, spacing=60, filter_period=0.5)] +
+        20 * [dict(volume_inj=1.5, duration_inj=6, spacing=120, filter_period=0.5)],
+    )
+
+
+binding_protocol = ITCProtocol(
+    '1:1 binding protocol',
+    sample_prep_method='Plates Standard.setup', # includes cleaning in the end
+    itc_method='ChoderaHSA20.inj',
+    analysis_method='Onesite',
+    experimental_conditions=dict(target_temperature=25, equilibration_time=300, stir_rate=1000, reference_power=5),
+    injections=[
+        dict(volume_inj=0.1, duration_inj=0.4, spacing=60, filter_period=0.5)] +
+        20 * [dict(volume_inj=1.5, duration_inj=6, spacing=120, filter_period=0.5)],
+    )
+
+
+cleaning_protocol = ITCProtocol(
+    'cleaning protocol',
+    sample_prep_method='Plates Clean.setup',
+    itc_method='ChoderaLHSA20.inj',          ## Is this correct or do we need ChoderaWaterWater5.inj here?
```

Comment:
I'd recommend water5inj.inj. It's the standard water into water protocol. Probably helpful for others if they need to debug after us.